lldb/source/Plugins/ExpressionParser/Swift/SwiftExpressionParser.cpp

```diff
@@ -1389,9 +1389,13 @@ SwiftExpressionParser::ParseAndImport(
   // GetImplicitImports.
   swift::performImportResolution(*source_file);
 
-  if (expr_diagnostics.HasErrors())
-    return make_error<ModuleImportError>(
-        llvm::toString(expr_diagnostics.GetAllErrors()));
+  if (expr_diagnostics.HasErrors()) {
+    // FIXME: This could be done more elegantly.
+    std::string msg = llvm::toString(expr_diagnostics.GetAllErrors());
+    if (StringRef(msg).contains(": could not build module "))
+      return make_error<ModuleImportError>(msg);
+    return llvm::createStringError(msg);
```

Comment:
nit: let's `std::move(msg)` here and above, since those are likely big strings.

Reply:
Is this not an optimization that Clang can do on its own?

Reply:
Nope, not in the middle of an expression. Clang will move when you have a naked return. E.g. imagine you had an `Error` class (which cannot be copied) instead of a string, this compiles:
```
return error;
```

but this doesn't compile:

```
return foo(error);
```